save-cloud-common/src/commonMain/kotlin/com/saveourtool/save/testsuite/TestSuitesSourceDto.kt

```diff
@@ -0,0 +1,30 @@
+package com.saveourtool.save.testsuite
+
+import com.saveourtool.save.entities.GitDto
+import kotlinx.serialization.Serializable
+
+typealias TestSuitesSourceDtoList = List<TestSuitesSourceDto>
+
+/**
+ * @param organizationName
```

Comment:
Shouldn't diktat trigger on it? We have both `@param` and `@property` for one field
Probably, related https://github.com/saveourtool/diktat/issues/1074

Reply:
actually it's my fault. I added `@param` manually, but it's `@property` -- diktat adds the proper annotation, but doesn't remove the wrong and even doesn't raise an issue

Reply:
added comment to https://github.com/saveourtool/diktat/issues/1074